src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
```

Comment:
You should avoid using the `*` selector as it impacts performance. It's better to style only the elements that require it. (CSS checklist)